lib/utils/testHelpers/repl.js

```diff
@@ -0,0 +1,2 @@
+// asset file for testing repl
```

Comment:
this file still needed?

Reply:
Yes, it helps testing the reRequire function in repl